config/application.rb

```diff
@@ -18,7 +18,7 @@ class Application < Rails::Application
 
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
-    config.i18n.default_locale = :'pt-BR'
+    config.i18n.default_locale = 'pt-BR'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.